moped-database/migrations/default/1751995587779_add_bicycle_signal_subcomponents/up.sql

```diff
@@ -0,0 +1,15 @@
+-- Add two new bicycle signal subcomponents for Signal - Traffic component
+INSERT INTO moped_subcomponents (subcomponent_name) VALUES 
+    ('Bicycle signal (may use pedestrian signal sign)'),
+    ('Bicycle signal (permissive conflicts)');
```

Comment:
Nathan reached out to me about a small change here for better clarity. Could you update to `Bicycle signal face (permissive conflicts)` before merging? Thank you!